packages/orchestrator/internal/metrics/meters.go

```diff
@@ -0,0 +1,75 @@
+package metrics
+
+import (
+	"go.opentelemetry.io/otel/metric"
+)
+
+type GaugeFloatType string
+type GaugeIntType string
+
+const (
+	SandboxCpuUsedGaugeName GaugeFloatType = "e2b.sandbox.cpu.used"
+)
+
+const (
+	SandboxRamUsedGaugeName  GaugeIntType = "e2b.sandbox.ram.used"
+	SandboxRamTotalGaugeName GaugeIntType = "e2b.sandbox.ram.total"
+	SandboxCpuTotalGaugeName GaugeIntType = "e2b.sandbox.cpu.total"
+)
+
+var gaugeFloatDesc = map[GaugeFloatType]string{
+	SandboxCpuUsedGaugeName: "Amount of CPU used by the sandbox.",
+}
+
+var gaugeFloatUnits = map[GaugeFloatType]string{
+	SandboxCpuUsedGaugeName: "{percent}",
+}
+
+var gaugeIntDesc = map[GaugeIntType]string{
+	SandboxRamUsedGaugeName:  "Amount of RAM used by the sandbox.",
+	SandboxRamTotalGaugeName: "Amount of RAM available to the sandbox.",
+	SandboxCpuTotalGaugeName: "Amount of CPU available to the sandbox.",
+}
+
+var gaugeIntUnits = map[GaugeIntType]string{
+	// TODO: is mebibyte small enough? shouldn't we just log bytes?
```

Comment:
The annoying part is, we will have to change envd...

Reply:
We are reporting `mem_total_mib` as `int64` from `envd`. If we want something more precise, we have to change envd